src-docs/src/views/accordion/accordion_grow.js

```diff
@@ -12,72 +9,48 @@ import {
 } from '../../../../src/components';
 import { htmlIdGenerator } from '../../../../src/services';
 
-class Rows extends Component {
-  state = {
-    counter: 1,
-  };
-
-  onIncrease() {
-    this.setState(prevState => ({
-      counter: prevState.counter + 1,
-    }));
+const Rows = () => {
+  const [counter, setCounter] = useState(1);
+  const rows = [];
+  for (let i = 1; i <= counter; i++) {
+    rows.push(<li key={i}>Row {i}</li>);
   }
-
-  onDecrease() {
-    this.setState(prevState => ({
-      counter: Math.max(0, prevState.counter - 1),
-    }));
-  }
-
-  render() {
-    const rows = [];
-    const { counter } = this.state;
-    for (let i = 1; i <= counter; i++) {
-      rows.push(<li key={i}>Row {i}</li>);
-    }
-    const growingAccordianDescriptionId = htmlIdGenerator()();
-    const listId = htmlIdGenerator()();
-    return (
-      <EuiText>
-        <EuiScreenReaderOnly>
-          <p id={growingAccordianDescriptionId}>
-            Currently height is set to {counter} items
-          </p>
-        </EuiScreenReaderOnly>
-        <EuiSpacer size="s" />
-        <p>
-          <EuiButton
-            onClick={() => this.onIncrease()}
-            aria-controls={listId}
-            aria-describedby={growingAccordianDescriptionId}>
-            Increase height to {counter + 1} items
-          </EuiButton>{' '}
-          <EuiButton
-            aria-controls={listId}
-            aria-describedby={growingAccordianDescriptionId}
-            onClick={() => this.onDecrease()}
-            isDisabled={counter === 1}>
-            Decrease height to {counter - 1} item{counter > 2 && 's'}
-          </EuiButton>
+  const growingAccordianDescriptionId = htmlIdGenerator()();
+  const listId = htmlIdGenerator()();
+  return (
+    <EuiText>
+      <EuiScreenReaderOnly>
+        <p id={growingAccordianDescriptionId}>
+          Currently height is set to {counter} items
         </p>
-        <ul id={listId}>{rows}</ul>
-      </EuiText>
-    );
-  }
-}
-
-class AccordionGrow extends Component {
-  render() {
-    return (
-      <EuiAccordion
-        id="accordion7"
-        buttonContent="Click me to toggle close / open"
-        initialIsOpen={true}
-        paddingSize="l">
-        <Rows />
-      </EuiAccordion>
-    );
-  }
-}
-
-export default AccordionGrow;
+      </EuiScreenReaderOnly>
+      <EuiSpacer size="s" />
+      <p>
+        <EuiButton
+          onClick={() => setCounter(counter + 1)}
+          aria-controls={listId}
+          aria-describedby={growingAccordianDescriptionId}>
+          Increase height to {counter + 1} items
+        </EuiButton>{' '}
+        <EuiButton
+          aria-controls={listId}
+          aria-describedby={growingAccordianDescriptionId}
+          onClick={() => setCounter(Math.max(0, counter - 1))}
+          isDisabled={counter === 1}>
+          Decrease height to {counter - 1} item{counter > 2 && 's'}
+        </EuiButton>
+      </p>
+      <ul i={listId}>{rows}</ul>
```

Comment:
```suggestion
      <ul id={listId}>{rows}</ul>
```

The a11y testing suite caught the mistake 🎉 This should get CI tests passing.